### ISSUE TITLE
Keeps saying, Custom app "beatsaber" not found when doing apply code

### ISSUE THREAD

**Issue by Miller30004** [kuba2k2/spicetify-beat-saber#21]
![image](https://github.com/user-attachments/assets/9f95d0c9-a0dc-4508-924d-7a1af5495530)
I know im doing something wrong, I just don't understand what

**Comment by kuba2k2** [comment]
The directory should be named "beatsaber", yours has a different name. Make sure you unpack the zip file correctly.

**Comment by Miller30004** [comment]
I'm trying to use an older version of beatsaber using bsmanager. Tried using both, renamed them, Just can't figure it out

**Comment by kuba2k2** [comment]
No. BeatSaber (the game) version doesn't matter. You don't need the shortcut to 1.29.1 in the spicetify folder.

The ZIP file you downloaded has a "beatsaber" directory inside. You need to unpack **this directory** to your spicetify apps folder. You probably used 7-zip or something, that created the `beatsaber-dist-2.3.0` folder for you. Just move the `beatsaber` folder from this one to the main spicetify folder.

**Comment by Miller30004** [comment]
ok i got it partially working but the  Beat Saber popup window doesn't open and does this little triangle 
![image](https://github.com/user-attachments/assets/f0158c14-0f14-4374-9dd1-bfad480d7890)
